hack/util.sh

```diff
@@ -47,16 +47,34 @@ function configure_os_server() {
 		export SERVER_HOSTNAME_LIST
 	fi
 
+        CREATE_MASTER_CERTS="openshift admin ca create-master-certs"
+        CREATE_NODE_CONFIG="openshift admin create-node-config"
+        CREATE_BOOTSTRAP_POLICY="openshift admin create-bootstrap-policy-file"
+        CREATE_CONFIG="openshift start"
+
+        CONFIG_VERSION=${1:-}
+        if [[ -n "${CONFIG_VERSION}" ]]; then
```

Comment:
And I *know* this is a super tiny nit-picky thing but RHEL RPMs don't have a Bash linter so we can't automate nit-picking on style. We've decided to use tabs for indenation and spaces for alignment in our scripts, same as in the Go code.